ugs-core/src/com/willwinder/universalgcodesender/uielements/panels/MachineStatusPanel.java

```diff
@@ -381,41 +382,35 @@ public void statusStringListener(ControllerStatus status) {
                 : (int) this.backend.getGcodeState().spindleSpeed;
         this.spindleSpeedValue.setText(Integer.toString(spindleSpeed));
     }
-    
-    private void setPostionValueColor(JLabel label, double newValue) {
+
+    private void trimLastSpace(StringBuilder sb) {
+        if (sb.length() > 0) sb.deleteCharAt(sb.length() - 1);
+    }
+
+    private void setPositionValueColor(JLabel label, double newValue) {
         if (!label.getText().equals(decimalFormatter.format(newValue))) {
-            label.setForeground(Color.red);
+            label.setForeground(ThemeColors.RED);
         } else {
-            label.setForeground(defaultTextColor);
+            label.setForeground(ThemeColors.LIGHT_BLUE);
         }
     }
 
     private void setStatusColorForState(String state) {
-        Color background = subPanelBackgroundColor;
-        Color foreground = defaultTextColor;
-        this.activeStatePanel.setGradientEnabled(true);
         if (backend.getSettings().isDisplayStateColor()) {
-            if (state.equals(Localization.getString("mainWindow.status.alarm"))
-                    || StringUtils.startsWithIgnoreCase(state, "Alarm")) {
-                background = new Color(128, 0, 0);
-                foreground = Color.WHITE;
-            } else if (state.equals(Localization.getString("mainWindow.status.hold"))) {
-                background = new Color(220, 200, 0);
-                foreground = Color.BLACK;
-            } else if (state.equals(Localization.getString("mainWindow.status.queue"))) {
-                background = new Color(220, 200, 0);
-                foreground = Color.BLACK;
-            } else if (state.equals(Localization.getString("mainWindow.status.run"))) {
-                background = new Color(0, 128, 0);
-                foreground = Color.WHITE;
-            } else {
-                this.activeStatePanel.setGradientEnabled(false);
+            Color background = ThemeColors.GREY;
```

Comment:
While you're fixing this could you update the `activeStateValueLabel` with the localized string?

Reply:
Oh snap, missed that one, thanks. 
Is this what you want me to fix?
```this.activeStateValueLabel.setText( status.getState() );```
What's the approach you use to change GRBL literals to translated values? Something like this?
```Localization.getString("mainWindow.status." + status.getState().toLowerCase()))```. I'm inclined to grab i18n values up front for performance reasons, probably use a `Map` of GRBL status to translated value.

Reply:
I would swap in the localizations right into these conditions. It looks like Jog might be missing, you can add that right into the english languange bundle. The online service I use for localization will synchronize terms later so no need to add it to every file.

Reply:
Marking as done for tracking my tasks.